DigitalLearningSolutions.Web/Styles/trackingSystem/centreConfiguration.scss

```diff
@@ -23,6 +23,27 @@
   }
 }
 
+.centre-detail--placeholder-image__downsize {
+  display: block;
+  width: 400px;
+  height: 200px;

```

Comment:
I feel like we must just be able to set the width and some other property so the height scales with it.

Reply:
Looks like I might be able to do something like
```
.class {
    $width: 400px;
    width: $width;
    height: 0.5 * width
}
```
Not sure if it is worth it though

Reply:
I think `height: auto` should just work

Reply:
That produces square 400x400 images by default, which is what we were trying to avoid by setting the height to less

Reply:
Ah sure, ignore me then